launch/sick_safetyscanners.launch

```diff
@@ -7,6 +7,7 @@
   <arg name="angle_start"           default="0" />
   <arg name="angle_end"             default="0" />
   <arg name="time_offset"           default="0.0" />
+  <arg name="min_intensities"       default="0.0" doc="minimal intensity for a laserscan point" />
```

Comment:
This should go into the readme as well for documentation

Reply:
c62042a